docs/source/changes.rst

```diff
@@ -1,3 +1,8 @@
+1.0.3
+=====
+* Fixed a bug in `pecan.hooks.HookController` for newer versions of Python3.4
```

Comment:
seems like this is not the case in this whole document, but for referencing classes and getting proper rendering double backticks should be used.